packages/cli/src/commands/list/index.ts

```diff
@@ -242,6 +257,8 @@ export default async function list(client: Client) {
 
   const headers = ['Age', 'Deployment', 'Status', 'Environment', 'Duration'];
   if (showUsername) headers.push('Username');
+  if (Object.keys(policy).length > 0)
+    headers.push(...['Expiration', 'Proposed Expiration']);
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** reduce columns

This makes the list very wide. Maybe we should remove the "Duration" and "Username" columns when using this policy preview.